<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditCalc</class>
 <widget class="QWidget" name="CreditCalc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>647</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Total amount (RUB)</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="totalAmount">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000000000000000.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>10000.000000000000000</double>
     </property>
     <property name="value">
      <double>1000.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Term (month)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QSpinBox" name="term">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>12</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Interest rate (%)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="interestRate">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <double>1000.000000000000000</double>
     </property>
     <property name="value">
      <double>3.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Type</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="annuity">
     <property name="text">
      <string>annuity</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QRadioButton" name="differentiated">
     <property name="text">
      <string>differentiated</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Result</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>monthly payment (RUB)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>overpayment on credit (RUB)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="monthlyPayment">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="overpayment">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="totalPayment">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>total payment (RUB)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QTextEdit" name="display"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
